src/TfaLoginPluginManager.php

```diff
@@ -26,9 +29,36 @@ class TfaLoginPluginManager extends DefaultPluginManager {
    *   The module handler.
    */
   public function __construct(\Traversable $namespaces, CacheBackendInterface $cache_backend, ModuleHandlerInterface $module_handler) {
-    parent::__construct('Plugin/TfaLogin', $namespaces, $module_handler, 'Drupal\tfa\TfaLoginInterface', 'Drupal\tfa\Annotation\TfaLogin');
+    parent::__construct('Plugin/TfaLogin', $namespaces, $module_handler, 'Drupal\tfa\Plugin\TfaLoginInterface', 'Drupal\tfa\Annotation\TfaLogin');
     $this->alterInfo('tfa_login_info');
     $this->setCacheBackend($cache_backend, 'tfa_login');
+    $this->tfaSettings = \Drupal::config('tfa.settings');
   }
 
+  /**
+   * Create an instance of a plugin.
+   * @param string $plugin_id
+   * @param array $configuration
+   * @return object
+   */
+  public function createInstance($plugin_id, array $configuration = array()) {
+    return parent::createInstance($plugin_id, $configuration); // TODO: Change the autogenerated stub
+  }
+
+  /**
+   * Returns an array of enabled login plugins.
+   * @param array $configuration
+   * @return array|null
+   */
+  public function getPlugins($configuration = array()){
```

Comment:
I think you actually use functions in the default plugin manager for getInstance to retrieve the plugins